src/ParaTest/Coverage/CoverageMerger.php

```diff
@@ -2,14 +2,19 @@
 
 namespace ParaTest\Coverage;
 
+use SebastianBergmann\CodeCoverage\CodeCoverage;
```

Comment:
interesting... so this does not cause an error if you don't have the latest CodeCoverage installed?